web/api/backend.go

```diff
@@ -66,6 +71,7 @@ func TokenHandler(w http.ResponseWriter, r *http.Request) {
 		w.WriteHeader(http.StatusInternalServerError)
 		return
 	}
+	w.WriteHeader(200)
```

Comment:
Not necessary. If it returns something back that is not a error, it will automatically be a StatusOK header

Reply:
👍